handlers/pipeline_test.go

```diff
@@ -0,0 +1,84 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/gaia-pipeline/gaia"
+	"github.com/gaia-pipeline/gaia/store"
+	hclog "github.com/hashicorp/go-hclog"
+	"github.com/labstack/echo"
+	"github.com/stretchr/testify/assert"
```

Comment:
It would be nice if you could remove this dependency. I know assert from other projects (docker for example) and they had at the end huge problems with it. It seems like it's not 100% reliable. :confused:

I know it's more overhead and makes the tests a bit more ugly but at least we are 100% using the standard library which I would like to stick if it's somehow possible. 🤗

Reply:
done